ferry.go

```diff
@@ -26,6 +26,7 @@ const (
 	StateWaitingForCutover = "wait-for-cutover"
 	StateCutover           = "cutover"
 	StateDone              = "done"
+	StateCanceled          = "canceled"
```

Comment:
Can we call it `Interrupted`/`Interruption`? Canceled makes it sound like the user changed their mind